process/project-lifecycle.md

```diff
@@ -174,7 +159,20 @@ To archive a project:
 * The proposal must remain open for at least 2 weeks of discussion after the maintainers are informed.
 * A vote must be finalized with 2/3 approval from the TAC or parent WG.
 
-#### Submission Forms to Archive
 
-1. [Proposal](TBD)
-2. [Legal](TBD)
+## Submission Process
+
+### Project creation or change of lifecycle stage
+
+For initiating the creation of a new project or for requesting a change of a project's lifecycle stage, an application must be submitted to the TAC. To this end, a maintainer of the project (or another interested party if no maintainer is available) creates a PR in this repository with the following changes:
+
+* A new file in the `project-lifecyle-documents` directory containing all information requested for a project creation or a lifecycle change review. This file must be based on the template for the respective lifecycle stage in the `templates` directory. The `PROJECT_NAME_` prefix of the template must be replaced by the project name.
```

Comment:
Is "the issue format" really different? Why can't the discussion happen in comment to the PR, just like on the issue?
The proposed workflow aims at minimizing the number of steps involved. Once the PR is approved it is merged and we're done. Otherwise we will need to get both the issue closed and then the PR submitted, approved, and merged as a second step.

Reply:
Discussion can certainly happen in a PR. 

I'm thinking of #101 where there's been robust discussion, and having that linear Issue format is helpful for readability. I'd prefer to design for accessible and digestible communication over time-to-approval on this one. 

cc @AevaOnline who might have thoughts from the CCC process as well.

Reply:
But if you comment at the bottom of a PR it is similarly linear, isn't it?

Reply:
So it depends on the lifecycle stage (within the CNCF).

For brand new projects applying to sandbox, its a form they fill out and we review.  This could be reasonably managed as an Issue template. But not all projects need to go through Sandbox, some can apply directly to incubation.

For Applying to Incubation, or changing from one stage to another (sandbox to incubation, incubation to graduation) its a PR. We keep [a template in the repo](https://github.com/cncf/toc/blob/main/process/graduation-proposal-template.md) that projects copy into their PR.  Discussion can occur on the PR in the particular content areas, but it still provides consistency in content. When accepted, they are merged in. Such as with [Linkerd](https://github.com/cncf/toc/pull/616), or [knative](https://github.com/cncf/toc/pull/762/files).

My recommendation: 
To be consistent in the processes and experience of projects, I would establish a template for each pursued stage and ask projects to copy the content into their PR to serve as the template. So one template for Sandbox, one for incubating, and one for graduation. The template should be clear in the information expected and its content sections should solicit meaningful information from the project that can drive an informed decision on whether they can move to the stage requested, or provide enough information to show why they cannot yet move. The deciding body should be able to clearly point to information in the PR, as well as other supporting materials, to justify their decision and provide recommendations actions back to the project based on that decision with a path forward.

Discussions then occur in the PR. The only outstanding process-oriented gotcha then would be the assignment of a sponsor to it. This should be mitigated by having the assigned or volunteered sponsor suggest a change to the PR for their handle in the otherwise empty "sponsor" section, and having the maintainer commit the suggestion.

Reply:
Thanks for the feedback, @TheFoxAtWork